database/index.js

```diff
@@ -53,5 +51,4 @@ const getLargeCarousel = (movieId, callback) => {
 
 
 module.exports.save = save;
-module.exports.getSmallCarousel = getSmallCarousel;
-module.exports.getLargeCarousel = getLargeCarousel;
+module.exports.getCarousel = getCarousel;
```

Comment:
I like this, I was about to ask why you had two separate queries :)

Reply:
Thank you! :)